src/_store/root.js

```diff
@@ -24,6 +24,25 @@ addLocaleData({
 
 const emptyObject = {};
 
+
+export const accountExclusion = async (token, accts = []) => {
```

Comment:
Can we improve the parameter name?

does token mean token to try next or?
what does accts means and how is it different from window.BinaryBoot.accounts